clouddriver-ecs/clouddriver-ecs.gradle

```diff
@@ -37,4 +51,24 @@ dependencies {
   testImplementation "org.objenesis:objenesis"
   testImplementation "org.spockframework:spock-core"
   testImplementation "org.spockframework:spock-spring"
+
+  integrationImplementation project(":clouddriver-web")
+  integrationImplementation "org.springframework:spring-test"
+  integrationImplementation "org.testcontainers:mysql"
+  integrationImplementation "mysql:mysql-connector-java"
+  integrationImplementation ("io.rest-assured:rest-assured:4.0.0") {
+    exclude group: "org.codehaus.groovy", module: "groovy" // use kork's version
+  }
+}
+
+task integrationTest(type: Test) {
+  description = "Runs Amazon ECS provider integration tests."
+  group = 'verification'
```

Comment:
What other groups are there? Can we make this an enum?

Reply:
AFAICT the only other places task groups are mentioned in `clouddriver` are in `build.gradle` and `clouddriver-kubernetes.gradle`, and this usage exactly matches the latter (also an integrationTest task). The [gradle docs](https://guides.gradle.org/writing-gradle-tasks/#add_a_task_description) also show using a string to specify task group, so I think this is the way to go?